DQMOffline/Trigger/python/HigPhotonJetHLTOfflineSource_cfi.py

```diff
@@ -28,7 +28,7 @@
     dirname = cms.untracked.string("HLT/Higgs/PhotonJet"), 
     verbose = cms.untracked.bool(False), # default: False
     triggerAccept = cms.untracked.bool(True), # default: True 
-    triggerResultsToken = cms.InputTag("TriggerResults"),
+    triggerResultsToken = cms.InputTag("TriggerResults","","HLT"),
```

Comment:
You are correct. Actually, every since we switched to the threaded framework, we've had code which attempted to catch these cases and throw and exception. It worked most of the time. Now that I'm implementing running all Paths concurrently within an Event I discovered that the checking code was incomplete. I now have a version of the checker which works (at least better) and it caught the cases which I fixed last week.

So we would have caught these cases more than a year ago if there hadn't been a bug in the checking code.